pyproject.toml

```diff
@@ -49,6 +49,12 @@
 [tool.black]
 line-length = 100
 
+[tool.isort]
+force_single_line = true
+honor_noqa = true
+lines_between_types = 1
```

Comment:
What is "types" in this context?

Reply:
```
import re
import subprocess
import sys

from collections import Counter
from collections import OrderedDict
from datetime import datetime

Reply:
Within the same section? Why is it needed then?

Reply:
the goal here was to minimize the number of changes in the repo, more than happy to revisit in the future, but leaving it here for now since the pattern is in place.

Reply:
Sounds like this line could use a context comment so that it doesn't force the future archeologists to guess what motivated its addition and if that judgment is still true.

Reply:
Why did you mark this as resolved? Nothing changed.

Reply:
darn, I didn't intend to.... was just buzzing through and must have accidentally.  Good call, will add comment!

Reply:
@webknjaz 

The line now include an explanation of what it does and why it is there

lines_between_types = 1 # Separate import/from with 1 line, minimizing changes at time of implementation

I see this as resolved, might you please click and resolve?

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback